### PR TITLE
Issue #4 - Update README

### LINKED ISSUE #4: Update README
> As a new developer to the soccerteam-app I need to know how to build and use the application.  Update the README to ensure any new devs can clone and build the app right away.

Acceptance Criteria:
1. Database info is added
2. maven commands to build and run the app
3. Additional software or dependencies needed besides what maven provides

### PR DESCRIPTION
**Pull request by davidbryan15**
ReadMe  updated with inital instructions. More to follow as App progresses

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
-# soccerteam-app
+# SoccerTeam-App
 Web app to organize soccer teams and their players
+
+## Requirements
+In order to run this app you must have the following technologies installed
+
+- MySql
+- Maven
```
Comment (pauldpearson) [inline README.md]: With the Maven wrapper you technically don't need to install Maven.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
-# soccerteam-app
+# SoccerTeam-App
 Web app to organize soccer teams and their players
+
+## Requirements
+In order to run this app you must have the following technologies installed
```
Comment (pauldpearson) [inline README.md:5]: How do you run the app?  Don't assume everyone will use IntelliJ, so what are the command line arguments needed to run your app?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,12 @@
...
+
+- MySql
```
Comment (pauldpearson) [inline README.md]: Is there a specific version of MySQL?

Comment (davidbryan15) [reply README.md]: I believe I have met your requirements in my latest commit.

File: README.md (hunk 2 above)
Comment (davidbryan15) [reply README.md:5]: I believe I have met your requirements in my latest commit.

**Pull request merged**